### ISSUE TITLE
table with 1 rows and 1 columns all over the place?

### ISSUE THREAD

**Issue by valiant8086** [nvaccess/nvda#15177]
### Steps to reproduce:
read some webpages or emails. I notice it in many emails that come from companies. Just this morning I experience it in an email from pay pal telling me that I have donated to a cherity and another one from Newegg advertising discounted refurbished iPads that easily demonstrate the problem.

Found another one from Amazon advertising the GPD Micro computer. It even has a table with 1 rows and 1 columns in it that says absolutely nothing at all inside it lol.

Probably will need to come up with a webpage that does it, not just an email? Hopefully I can edit this issue if I find one. I should also verify the issue is present in the latest version of Betterbird with the same email message.

### Actual behavior:
NVDA says "out of table table with 1 rows and 1 columns table with 1 rows and 1 columns" and similar things in a manner that seems excessive and reduces productivity.

### Expected behavior:
I do feel like this problem was not here before, but I can't identify the difference outright. It makes me want to turn off table announcements, but then I miss ones that I need to know about and actually have use for. I expect that NVDA used to not identify so many things as tables perhaps, and I'd like to have the idea of cutting back on this investigated. At the very least we could be ignoring the 1 rows and 1 columns thing, as there's no navigating a 1 row and 1 column table using table navigation. Not announcing that nonsense would of course mean we don't know that t could navigate to this, but I'm willing to sacrifice that if others are.

### NVDA logs, crash dumps and other attachments:
Here is an example of the newegg message recorded in mp3.
[table with 1 columns and 1 rows.zip](https://github.com/nvaccess/nvda/files/12138804/table.with.1.columns.and.1.rows.zip)

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
alpha-28712,9157916f (2023.2.0.28712)

#### Windows version:
10.0.19045 Build 19045

#### Name and version of other software in use when reproducing the issue:
Thunderbird 60.9.0 (32-bit)  portable from portableapps.com

#### Other information about your system:
i7 12700k desktop, integrated GPU only, 32gb RAM. I have JAWS installed also.

#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no

**Comment by valiant8086** [comment]
Lol it does it right here on the Github page, on the heading 3 that says steps to reproduce in my initial posting, it says "table with 1 rows and 1 columns heading level 3 steps to reproduce"

I didn't find the end of that table, I think the entire post is the element.

**Comment by mzanm** [comment]
Hi.
You might have the option to include layout tables checked, press NVDA + N, go to settings, and then to browse mode, and make sure the 'include layout tables' checkbox is unchecked.

**Comment by Brian1Gaff** [comment]
I've seen that as far back as the last version of nvda, In my view it seems 
that the emails are the problem here, whatever is writing the html is making 
a complete mess of it It is mostly companies, might be worth looking at 
headers to see what email system is doing it. I found one company altering 
the html but not the text version resulting in the display of the text being 
old compared with the html. I sometimes wonder if anyone checks  bulk 
emails. The worst of these came from Transport for London, and I wrote to 
the  pr department and it stopped.
 Good luck with Amazon etc though, I suspect they have a pr company doing 
all that stuff.
 Brian

-- 
***@***.***
Sent via blueyonder.(Virgin media)
Please address personal E-mail to:-
***@***.***, putting 'Brian Gaff'
in the display name field.
----- Original Message ----- 
From: "valiant8086" ***@***.***>
To: "nvaccess/nvda" ***@***.***>
Cc: "Subscribed" ***@***.***>
Sent: Sunday, July 23, 2023 2:06 PM
Subject: [nvaccess/nvda] table with 1 rows and 1 columns all over the place? 
(Issue #15177)

**Comment by Brian1Gaff** [comment]
These do tend to be marked up as tables though, does your suggestion affect 
other sized tables?
 Brian

-- 
***@***.***
Sent via blueyonder.(Virgin media)
Please address personal E-mail to:-
***@***.***, putting 'Brian Gaff'
in the display name field.
----- Original Message -----

**Comment by amirsol81** [comment]
@valiant8086 As frequent or obvious it might sound or be, you should provide a link in order for everyone to duplicate the issue easily. Saying that it happens frequently with emails might not help much.

**Comment by valiant8086** [comment]
I have received an email ad from TP Link that has a link to view the email in browser. Following that link gives me the same issue, although in this case it's not really table with 1 rows and 1 columns, it's about 6 nested tables in some cases and more than 2 that have no purpose other than to slow-down speech. I mean, it'll say table with 1 rows and 6 columns table with 1 rows and 6 columns. Then it'll have some information and say, out of table out of table table with 1 rows and 6 columns, and then have some more information, etc. This one is especially tiresome. Maybe there's not a good solution for this except perhaps if I can set it to read the entering and leaving tables etc messages after the text instead of before it. Going to put my ad link tyhat demonstrates what I'm talking about here, but I am unsure if it'll let others follow it.

https://us11.campaign-archive.com/?e=8202699722&u=3401cfb78213d0a7443178702&id=2f418ae38f

**Comment by ABuffEr** [comment]
Hi @valiant8086,
did you tried to disable reading of layout tables, as suggested a month ago?

**Comment by valiant8086** [comment]
Sorry, I turned it off and I actually thought I already commented on this saying that it didn't change anything. However, I just turned it off again, this is a reinstalled copy of NVDA from my portable version because my ssd died and I had to reinstall everything, and it helped in a specific example I had just encountered which was particularly bad, and the above linked example as well. I'll leave this open for the time being, since I was still experiencing it after I turned that off on the previous install.
